src/dataflow/src/logging/timely.rs

```diff
@@ -287,25 +287,29 @@ pub fn construct<A: Allocate>(
 
         let operates = operates.as_collection();
 
+        // Feedback delay for log flushing. This should be large enough that there is not a tight
+        // feedback loop that prevents high-throughput work, but not so large that we leave volumes
+        // of logging data around that we do not need. Effectively, a throughput v memory trade-off.
+        let delay = std::time::Duration::from_nanos(10_000_000_000);
```

Comment:
no real reason - following a long standing mcsherry precent of expressing time in nanos in dataflow code :)

Reply:
I think @ruchirK got it from me, and I used it because parts of timely run at sub-microsecond level, and so I just do nanoseconds everywhere out of habit. No reason this needs to be bigger than a second. One could push for millisecond, as that is our timestamp unit, but imo we should change that too.

Is it stressful as nanos, or just unclear why so complicated?

Reply:
It's not that it was complicated but rather my line of inquiry went like this:

1. Huh, the underscores kind of make that human readable, if you know that there are 1 billion nanos per second. But it still takes some effort to read.
2. In a previous life, we always use helpers to convert from human readable to what the API wanted. For example, doing something like `from_nanos(10 * NSEC_PER_SEC)` to make it easier to humans to read. I wonder if we should do something like that.
3. Constants are annoying. I wonder if the `duration` library has a `from_secs` method. It does! I wonder why they didn't use that -- I should ask!